jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/TemplateSettings.kt

```diff
@@ -38,15 +36,14 @@ class TemplateSettings(val project: Project) {
         private set
     lateinit var function: JComboBox<Function>
         private set
-    lateinit var runtime: JComboBox<Runtime>
-        private set
     lateinit var pathMappingsTable: PathMappingsComponent
         private set
     private lateinit var functionModels: DefaultComboBoxModel<Function>
     private lateinit var imageSettingsPanel: JPanel
-    private lateinit var runtimeModel: SortedComboBoxModel<Runtime>
     lateinit var environmentVariables: EnvironmentVariablesTextField
         private set
+    private lateinit var imageDebuggerModel: SortedComboBoxModel<ImageDebugSupport>
```

Comment:
is this panel image-only? what happened to the zip runtimes?

Reply:
This panel is image only, for zip runtimes we pull from the template